src/xcode/ENA/ENA/Source/Services/Event Registration/QRCodePlayground.swift

```diff
@@ -0,0 +1,40 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import Base32
+
+// Since this is a POC this enum exists. Content should be moved into model/service that will later be used.
+enum QRCodePlayground {
+	
+	static func generateQRCode(with string: String, size: CGSize = CGSize(width: 400, height: 400)) -> UIImage? {
```

Comment:
Also here, any chance of having a unit test for that?

Reply:
Maybe [read back](https://stackoverflow.com/a/49275021/194585) in the generated QR code and verify the String?

Reply:
@whiskey that was a nice Idea, please check my updated Tests.